pkg/processor/sourceprocessor/source_processor.go

```diff
@@ -21,11 +21,10 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/sourceprocessor/observability"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/model/pdata"
```

Comment:
This I'd actually try to leave as is and try to enforce it this way. IMHO it's a nice grouping pattern to group `go.opentelemetry.io` separately. We should add `impi` to CI

```suggestion
	"go.opentelemetry.io/collector/component"
	"go.opentelemetry.io/collector/consumer"
	"go.opentelemetry.io/collector/model/pdata"
	
	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/sourceprocessor/observability"
```

Reply:
I agree, this is nicer, but some checks are failing for this grouping as they try to enforce 3 groups and all of above are treated as 3rd party imports

Reply:
```
➜  sourceprocessor git:(drosiek-source-processor) golangci-lint run \
        --enable deadcode,unused,goimports \
        --allow-parallel-runners \
        --max-same-issues 10 \

source_processor.go:27: File is not `gofmt`-ed with `-s` (gofmt)
```